osmnx/pois.py

```diff
@@ -372,21 +443,33 @@ def pois_from_point(point, distance=None, amenities=None):
         a lat-long point
     distance : numeric
         distance in meters
+    tags : dict
+        Dictionary of tag keys and values that will be used for finding POIs in the selected area.
+        Keys may be strings or lists of strings.
+        Values make be string, lists of strings, or None, if all values should be returned for a given key.
+        By default, all POIs with an 'amenity' key of any value will be be returned.
+
+    Keyword Arguments
+    -----------------
     amenities : list
-        List of amenities that will be used for finding the POIs from the selected area. 
-        See available amenities from: http://wiki.openstreetmap.org/wiki/Key:amenity
+        If provided, will override tags with the dictionary {'amenities':amenities}.
+        Accommodates the depricatd amenities parameter.
 
     Returns
     -------
     GeoDataFrame 
     """
+    if 'amenities' in locals():
```

Comment:
Is this simpler than just leaving default argument `amenities=None` and checking `if amenities is not None` here? (Same question goes for the other pois_from_xxxx() functions.)